src/App.js

```diff
@@ -1,16 +1,69 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import Landing from './components/Landing/Landing'
+import Landing from './components/Landing/Landing';
 import Store from './components/Store/Store';
-import './index.css'
+import Cart from './components/Cart/Cart';
+import './index.css';
 
 const App = () => {
+  const [cartVisible, setCartVisible] = React.useState(false);
+  const [blur, setBlur] = React.useState(false);
+  const [cartContent, setCartContent] = React.useState([]);
+
+  const openCart = () => {
+    setCartVisible(true);
+    setBlur(true);
+  };
+
+  const closeCart = () => {
+    setCartVisible(false);
+    setBlur(false);
+  };
+
+  const addToCart = (game) => {
+    setCartContent((prevContent) => [
+      { name: game.name, price: game.price },
+      ...prevContent,
+    ]);
+  };
+
+  const removeGame = (game) => {
+    setCartContent((prevContent) =>
+      prevContent.filter((iteratedGame) => iteratedGame.name !== game.name)
+    );
+  };
+
   return (
     <div className='app'>
+      <Cart
+        cartVisible={cartVisible}
```

Comment:
You should try to name boolean variables using `is` or `should` smth like `isCartVisible` or `shouldCartBeVisible`. It's easier for other readers to understand that that's a boolean value.